### ISSUE TITLE
Unexpected conversion

### ISSUE THREAD

**Issue by cmft** [hgrecco/pint#586]
Hi,
I created a `quantity` for `Sv`  (Sievert) and I got  `gray`, this is not done for other non-SI units. See the attached code:

```
from pint import UnitRegistry
UR = UnitRegistry()
q = UR.Quantity("1 Sv")
print q
```
IMO  the  conversion should not be done automatically.

**Comment by cpascual** [comment]
Sv is an operational unit for equivalent dose and effective dose. It is not a physical quantity
Gy is the physical quantity for absorbed dose.

The effective dose (in Sv) is calculated from the absorbed dose (Gy) after applying a set of factors which are quite subjective (furthermore, the definition of Sv is unclear for large doses, since the quality factors are not well defined in that case). This cannot be automated by a package such as pint. 

Pint deals with this [by defining Sv=Gy](https://github.com/hgrecco/pint/blob/master/pint/default_en.txt), but since this is only true in some specific cases, **IMHO pint should define Gy and Sv as different (and, ideally, incompatible) units.**

For more details, see [this thread](https://www.researchgate.net/post/What_is_the_difference_between_Sievert_and_Gray_A_practical_question_concerning_the_SI_units_for_ionizing_radiation) and, specifically this post:

> Jerry J Battista
The University of Western Ontario
Allow me to clarify....
Absorbed Dose (Gray) is  purely Physical  as measured by radiation dosimeters (e.g. ion chamber, diode, TLD, film,  etc). The unit of  1 Gray = 1 J/kg - is simply the concentration of physical energy absorbed locally from the radiation exposure.
Effective Dose (Sv) wasintroduced to help incorporate the Biological impact of radiation but it still relies on the same fundamental dimensional unit (J/kg). It is the Physical Dose (in Gy) multiplied by unitless factors, such as Radiation Weights to account for different types (LET, RBE) of radiation, and/or Tissue/Organ Weights to account for differential radiosensitivity/importance of each organ to human survival. Dose rate corrections can also be applied. The origin of the organ weighting factors is poorly explained in ICRP documents. They are rough estimates, in my view, with unclear justification. 
Effective Dose (Sv) - The individual effective organ doses (in Sv) are summed together in a weighted fashion to yield a virtual whole-body Effective Dose. The Effective Dose is intended to match "overall risk" of a partial body non-uniform radiation exposure to a whole body uniform exposure.
Risk Estimation- The Effective Dose is often multiplied by risk-conversion factors derived from Japanese survivors with whole body exposures (e.g 5% per Sv of Effective Dose) to estimate the probability of a radiation -induced cancer death. Again  this is just an estimate for a similarly exposed population - not to be used for an individual exposed person.
In brief, dose (Gy)  is easy to measure physically...Effective Dose is much more complex, debatable,  and requires 'biological inputs' with very large uncertainties.
Radiation Protection is not (yet) at the accuracy level of Radiation Dosimetry !
I hope this helps...

**Comment by hgrecco** [comment]
Maybe a context can be used. If somebody wants to champion this, it will be welcomed. Until then, I am closing this for lack of interest.

**Comment by danieldjewell** [comment]
I can understand wanting to close thing to clean up - but I really think that this should be left open as the functionality, as currently implemented in Pint, is not correct and could produce misleading results. Just because of a "lack of interest" doesn't change the operational matter that the behavior *ought* to be fixed. (Mark it as Todo or whatever - but it's still an open issue.) 

As noted by @cpascual, although the units of 1 Gy and 1 Sv are "J/kg" (or if you prefer "J &middot; kg<sup>-1</sup>") -- and similarly for rad/rem, 1 Gy = 1 Sv only if the weighing factors are 1. 

[ICRP Publication 103](http://www.icrp.org/publication.asp?id=ICRP%20Publication%20103) [[PDF]](http://journals.sagepub.com/doi/pdf/10.1177/ANIB_37_2-4) is one of the guiding documents on this. It's not a short or easy read, but important reference points are: 
- Table 2 on PDF page 68 (weighing factors for radiation types) 
- Table 3 on PDF page 69 (tissue weighing factors) 
- Section 4.3.4 "Effective dose and tissue weighing factors" on PDF page 72

The equation given on page 72 is (and this doesn't even take into account *reference phantoms* and averages of both the *Male* and *Female* averages) : 

![E = \sum_{T}w_{T}H_{T} = \sum_{T}w_{T}\sum_{R}w_{R}D_{T,R}](https://latex.codecogs.com/svg.latex?E%20%3D%20%5Csum_%7BT%7Dw_%7BT%7DH_%7BT%7D%20%3D%20%5Csum_%7BT%7Dw_%7BT%7D%5Csum_%7BR%7Dw_%7BR%7DD_%7BT%2CR%7D) 

All things equal, a good example is just on the basis of radiation type. If 1 Gy of &gamma;/x-ray/electrons is delivered to the whole body then yes, it more or less works out to 1 Sv. 

However, if 1 Gy of &alpha; particles (or "fission fragments or heavy ions") is delivered, the weighing factor becomes **20** meaning that the *Equivalent dose* is ***20 Sv*** (all other things being equal). 

Perhaps, most importantly, the [Current BIPM SI System Brochure-pdf](https://www.bipm.org/utils/common/pdf/si-brochure/SI-Brochure-9.pdf) adopted several measures regarding this topic: 
- ["Concerning the sievert" on PDF page 178](https://www.bipm.org/utils/common/pdf/si-brochure/SI-Brochure-9.pdf#page=178) reiterating a simpler equation of ```H = Q * N * D``` where **H** is the **dose equivalent** and **D** is the **absorbed dose** (and Q/N are the weighing factors) stating, in part: ```Thus, for a given radiation, the numerical value of H in joules per kilogram may differ from that of D in  joules  per  kilogram  depending  upon  the  values  of  Q and N. In  order  to  avoid  any  risk  of  confusion  between  the  absorbed  dose  D and  the  dose  equivalent  H,  the  special  names  for  the  respective  units  should  be  used,  that  is,  the  name  gray  should  be  used  instead  of  joules  per  kilogram for the unit of absorbed dose D and the name sievert instead of joules per kilogram for the unit of dose equivalent H.```
- ["Dose equivalent" on PDF page 184](https://www.bipm.org/utils/common/pdf/si-brochure/SI-Brochure-9.pdf#page=184) revising the previous statement and stating very officially: 

---
The  quantity  dose  equivalent  H  is  the  product  of  the  absorbed  dose  D  of  ionizing  radiation  and  the dimensionless factor Q (quality factor) defined as a function of linear energy transfer by the ICRU:

![H=Q \cdot D](https://latex.codecogs.com/svg.latex?H%3DQ%20%5Ccdot%20D) 

Thus, for a given radiation, the numerical value of H in joules per kilogram may differ from that of D in joules per kilogram depending on the value of Q.

*The Committee further decides to maintain the final sentence in the explanation as follows:*

In order to ***avoid any risk of confusion*** between the absorbed dose D and the dose equivalent H, **the special names for the respective units should be used**, that is, the name ***gray*** should be used instead  of  joules  per  kilogram  ***for  the  unit  of  absorbed  dose  D*** and  the  name ***sievert***  instead  of  joules per kilogram for the ***unit of dose equivalent H***. [emphasis added]

--- 

In other words, there could be *severe* ramifications (like life-or-death *radiation overdose* consequences) if someone were to rely on Pint to try converting Gray to Sieverts. One would hope that this would never happen, but just to take the point to it's extreme... 

TL;DR;: ```1 Gy === 1 Sv``` *if and only if* the weighing factors are 1. The Gray is the unit of **absorbed dose** and the Sievert is the unit of **dose equivalent**. They are not the same. Making the assumption they are could have severe life-impacting consequences if someone were to rely on that.
